src/TestUtils/src/DeviceTests/AssertionExtensions.Android.cs

```diff
@@ -176,10 +174,20 @@ public static async Task HideKeyboardForView(this AView view, int timeout = 1000
 
 			try
 			{
-				if (!KeyboardManager.HideKeyboard(view))
-					throw new Exception("HideKeyboard operation failed");
-
-				await view.WaitForKeyboardToHide(timeout);
+				int retry = 2;
+				for (int i = 0; i < retry; i++)
+				{
+					KeyboardManager.HideKeyboard(view);
+					try
+					{
+						await view.WaitForKeyboardToHide(timeout);
```

Comment:
Always retry with the same timeout? Shall we increase in every iteration?

Reply:
reworked to use are `wait` method that already has backoff behavior.